WeightProducer/src/WeightProducer.cc

```diff
@@ -555,19 +621,37 @@ std::vector<double> WeightProducer::generateWeights(PUScenario sc, const TH1* da
       5.005E-06};
     npuProbs = npuSummer12_S10;
   }
-
+  if(sc==Spring15) nMaxPU=25;
   std::vector<double> result(nMaxPU);
+
   double s = 0.0;
+if(sc==Spring15){
```

Comment:
Fixed everything below, but if there are other edits you recommend you can keep sending them my way. I will wait for a bit before making another pull request. 
-Rishi
> On Oct 5, 2015, at 5:13 PM, Kevin Pedro <notifications@github.com> wrote:
> 
> In WeightProducer/src/WeightProducer.cc <https://github.com/TreeMaker/TreeMaker/pull/68#discussion_r41155542>:
> 
> >    double s = 0.0;
> > +if(sc==Spring15){
> Why is this here if it's all commented out?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/TreeMaker/TreeMaker/pull/68/files#r41155542>.
>